src/content/en/tools/workbox/guides/storage-quota.md

```diff
@@ -19,13 +19,13 @@ When setting up a route and runtime caching strategy, you can add in an instance
 [`workbox.expiration.Plugin`](/web/tools/workbox/reference-docs/latest/workbox.expiration.Plugin)
 configured with settings that make the most sense for the type of assets you're caching.
 
-For instance, the following configuration might be use for caching images at runtime, with both
+For instance, the following configuration might be used for caching images at runtime, with both
 explicit limits as well as automatic cleanup if quota is exceeded:
 
 ```javascript
 workbox.routing.registerRoute(
   // Match common image extensions.
-  new RegExp('\.(?:png|gif|jpg|svg)$'),
+  new RegExp('\\.(?:png|gif|jpg|jpeg|svg)$'),
```

Comment:
@jeffposnick - can you take a look at this regex update? If you're happy, please approve the PR and merge.